### PR TITLE
Add an `antd@4` example with `next@9.2`

### LINKED ISSUE #9830: Update Ant Design Examples for New CSS Support (9.2)
# import ant-design library throw error

can you provide an example import ant-design use [RFC] css support? thanks.
when i use [RFC] css support and import ant-design library, it throw error :
```
./node_modules/antd/lib/button/style/index.css
Global CSS cannot be imported from files other than your Custom <App>. Please move all global CSS imports to pages/_app.js.
```
here is my next-config.js 

```
module.exports = {
    experimental: {
      css: true
    },
  }
```

### PR DESCRIPTION
**Pull request by Pegase745**
This is an example using [antd@4](https://next.ant.design/) version, and `next@9.2.x`.

As you can see in the screenshot, I am using its components and even overriding them, without the use of `next/css` plugin.

***

**Note** One detail though, when loading a page the first time, I experience a blink on DEV mode, where I see my styles applied and then disappear (it may be related to https://github.com/zeit/next.js/issues/10148, but not sure).

Otherwise when,
 - you modify CSS and hot reload,
 - or build and start

there are no issues.

Fixes https://github.com/zeit/next.js/issues/9830

### CODE REVIEW

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | zeit/next.js canary  | Pegase745/next.js example/antd-4-next-9.2 | Change |
| - | - | - | - |
| buildDuration | 11.7s | 11.7s | -24ms |
| nodeModulesSize | 52.6 MB | 52.6 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | Pegase745/next.js example/antd-4-next-9.2 | Change |
| - | - | - | - |
| main-HASH.js gzip | 5.12 kB | 5.12 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| 4952ddcd88e7..54d3.js gzip | 4.68 kB | 4.68 kB | ✓ |
| commons.HASH.js gzip | 4.06 kB | 4.06 kB | ✓ |
| de003c3a9d30..b50f.js gzip | 13.6 kB | 13.6 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 67.3 kB | 67.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | Pegase745/next.js example/antd-4-next-9.2 | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.12 kB | 4.12 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| 4952ddcd88e7..dule.js gzip | 5.56 kB | 5.56 kB | ✓ |
| de003c3a9d30..dule.js gzip | 12.4 kB | 12.4 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 61.9 kB | 61.9 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | Pegase745/next.js example/antd-4-next-9.2 | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 4.76 kB | 4.76 kB | ✓ |
| Overall change | 4.76 kB | 4.76 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | Pegase745/next.js example/antd-4-next-9.2 | Change |
| - | - | - | - |
| _app.js gzip | 1.15 kB | 1.15 kB | ✓ |
| _error.js gzip | 4.07 kB | 4.07 kB | ✓ |
| hooks.js gzip | 779 B | 779 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.89 kB | 2.89 kB | ✓ |
| routerDirect.js gzip | 283 B | 283 B | ✓ |
| withRouter.js gzip | 282 B | 282 B | ✓ |
| Overall change | 9.68 kB | 9.68 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | Pegase745/next.js example/antd-4-next-9.2 | Change |
| - | - | - | - |
| _app.module.js gzip | 576 B | 576 B | ✓ |
| _error.module.js gzip | 3.06 kB | 3.06 kB | ✓ |
| hooks.module.js gzip | 371 B | 371 B | ✓ |
| index.module.js gzip | 212 B | 212 B | ✓ |
| link.module.js gzip | 2.46 kB | 2.46 kB | ✓ |
| routerDirect..dule.js gzip | 273 B | 273 B | ✓ |
| withRouter.m..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 7.22 kB | 7.22 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | Pegase745/next.js example/antd-4-next-9.2 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | zeit/next.js canary  | Pegase745/next.js example/antd-4-next-9.2 | Change |
| - | - | - | - |
| index.html gzip | 1.02 kB | 1.02 kB | ✓ |
| link.html gzip | 1.03 kB | 1.03 kB | ✓ |
| withRouter.html gzip | 1.02 kB | 1.02 kB | ✓ |
| Overall change | 3.07 kB | 3.07 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | zeit/next.js canary  | Pegase745/next.js example/antd-4-next-9.2 | Change |
| - | - | - | - |
| buildDuration | 12.4s | 12s | -405ms |
| nodeModulesSize | 52.6 MB | 52.6 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | Pegase745/next.js example/antd-4-next-9.2 | Change |
| - | - | - | - |
| main-HASH.js gzip | 5.12 kB | 5.12 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| 4952ddcd88e7..54d3.js gzip | 4.68 kB | 4.68 kB | ✓ |
| commons.HASH.js gzip | 4.06 kB | 4.06 kB | ✓ |
| de003c3a9d30..b50f.js gzip | 13.6 kB | 13.6 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 67.3 kB | 67.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | Pegase745/next.js example/antd-4-next-9.2 | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.12 kB | 4.12 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| 4952ddcd88e7..dule.js gzip | 5.56 kB | 5.56 kB | ✓ |
| de003c3a9d30..dule.js gzip | 12.4 kB | 12.4 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 61.9 kB | 61.9 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | Pegase745/next.js example/antd-4-next-9.2 | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 4.76 kB | 4.76 kB | ✓ |
| Overall change | 4.76 kB | 4.76 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | Pegase745/next.js example/antd-4-next-9.2 | Change |
| - | - | - | - |
| _app.js gzip | 1.15 kB | 1.15 kB | ✓ |
| _error.js gzip | 4.07 kB | 4.07 kB | ✓ |
| hooks.js gzip | 779 B | 779 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.89 kB | 2.89 kB | ✓ |
| routerDirect.js gzip | 283 B | 283 B | ✓ |
| withRouter.js gzip | 282 B | 282 B | ✓ |
| Overall change | 9.68 kB | 9.68 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | Pegase745/next.js example/antd-4-next-9.2 | Change |
| - | - | - | - |
| _app.module.js gzip | 576 B | 576 B | ✓ |
| _error.module.js gzip | 3.06 kB | 3.06 kB | ✓ |
| hooks.module.js gzip | 371 B | 371 B | ✓ |
| index.module.js gzip | 212 B | 212 B | ✓ |
| link.module.js gzip | 2.46 kB | 2.46 kB | ✓ |
| routerDirect..dule.js gzip | 273 B | 273 B | ✓ |
| withRouter.m..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 7.22 kB | 7.22 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | Pegase745/next.js example/antd-4-next-9.2 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | zeit/next.js canary  | Pegase745/next.js example/antd-4-next-9.2 | Change |
| - | - | - | - |
| _error.js gzip | 46.5 kB | 46.5 kB | ✓ |
| 404.html gzip | 1.43 kB | 1.43 kB | ✓ |
| hooks.html gzip | 1.07 kB | 1.07 kB | ✓ |
| index.js gzip | 46.5 kB | 46.5 kB | ✓ |
| link.js gzip | 72 kB | 72 kB | ✓ |
| routerDirect.js gzip | 70.2 kB | 70.2 kB | ✓ |
| withRouter.js gzip | 70 kB | 70 kB | ✓ |
| Overall change | 308 kB | 308 kB | ✓ |

</details>

</details>


Commit: 4950873a824fdcc0f6f850894bbda3c5b7adb2a2

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | zeit/next.js canary  | Pegase745/next.js example/antd-4-next-9.2 | Change |
| - | - | - | - |
| buildDuration | 11.6s | 11.7s | ⚠️ +60ms |
| nodeModulesSize | 52.6 MB | 52.6 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | Pegase745/next.js example/antd-4-next-9.2 | Change |
| - | - | - | - |
| main-HASH.js gzip | 5.12 kB | 5.12 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| 4952ddcd88e7..54d3.js gzip | 4.68 kB | 4.68 kB | ✓ |
| commons.HASH.js gzip | 4.06 kB | 4.06 kB | ✓ |
| de003c3a9d30..b50f.js gzip | 13.6 kB | 13.6 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 67.3 kB | 67.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | Pegase745/next.js example/antd-4-next-9.2 | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.12 kB | 4.12 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| 4952ddcd88e7..dule.js gzip | 5.56 kB | 5.56 kB | ✓ |
| de003c3a9d30..dule.js gzip | 12.4 kB | 12.4 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 61.9 kB | 61.9 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | Pegase745/next.js example/antd-4-next-9.2 | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 4.76 kB | 4.76 kB | ✓ |
| Overall change | 4.76 kB | 4.76 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | Pegase745/next.js example/antd-4-next-9.2 | Change |
| - | - | - | - |
| _app.js gzip | 1.15 kB | 1.15 kB | ✓ |
| _error.js gzip | 4.07 kB | 4.07 kB | ✓ |
| hooks.js gzip | 779 B | 779 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.89 kB | 2.89 kB | ✓ |
| routerDirect.js gzip | 283 B | 283 B | ✓ |
| withRouter.js gzip | 282 B | 282 B | ✓ |
| Overall change | 9.68 kB | 9.68 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | Pegase745/next.js example/antd-4-next-9.2 | Change |
| - | - | - | - |
| _app.module.js gzip | 576 B | 576 B | ✓ |
| _error.module.js gzip | 3.06 kB | 3.06 kB | ✓ |
| hooks.module.js gzip | 371 B | 371 B | ✓ |
| index.module.js gzip | 212 B | 212 B | ✓ |
| link.module.js gzip | 2.46 kB | 2.46 kB | ✓ |
| routerDirect..dule.js gzip | 273 B | 273 B | ✓ |
| withRouter.m..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 7.22 kB | 7.22 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | Pegase745/next.js example/antd-4-next-9.2 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | zeit/next.js canary  | Pegase745/next.js example/antd-4-next-9.2 | Change |
| - | - | - | - |
| index.html gzip | 1.02 kB | 1.02 kB | ✓ |
| link.html gzip | 1.03 kB | 1.03 kB | ✓ |
| withRouter.html gzip | 1.02 kB | 1.02 kB | ✓ |
| Overall change | 3.07 kB | 3.07 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | zeit/next.js canary  | Pegase745/next.js example/antd-4-next-9.2 | Change |
| - | - | - | - |
| buildDuration | 12s | 12.2s | ⚠️ +120ms |
| nodeModulesSize | 52.6 MB | 52.6 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | Pegase745/next.js example/antd-4-next-9.2 | Change |
| - | - | - | - |
| main-HASH.js gzip | 5.12 kB | 5.12 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| 4952ddcd88e7..54d3.js gzip | 4.68 kB | 4.68 kB | ✓ |
| commons.HASH.js gzip | 4.06 kB | 4.06 kB | ✓ |
| de003c3a9d30..b50f.js gzip | 13.6 kB | 13.6 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 67.3 kB | 67.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | Pegase745/next.js example/antd-4-next-9.2 | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.12 kB | 4.12 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| 4952ddcd88e7..dule.js gzip | 5.56 kB | 5.56 kB | ✓ |
| de003c3a9d30..dule.js gzip | 12.4 kB | 12.4 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 61.9 kB | 61.9 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | Pegase745/next.js example/antd-4-next-9.2 | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 4.76 kB | 4.76 kB | ✓ |
| Overall change | 4.76 kB | 4.76 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | Pegase745/next.js example/antd-4-next-9.2 | Change |
| - | - | - | - |
| _app.js gzip | 1.15 kB | 1.15 kB | ✓ |
| _error.js gzip | 4.07 kB | 4.07 kB | ✓ |
| hooks.js gzip | 779 B | 779 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.89 kB | 2.89 kB | ✓ |
| routerDirect.js gzip | 283 B | 283 B | ✓ |
| withRouter.js gzip | 282 B | 282 B | ✓ |
| Overall change | 9.68 kB | 9.68 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | Pegase745/next.js example/antd-4-next-9.2 | Change |
| - | - | - | - |
| _app.module.js gzip | 576 B | 576 B | ✓ |
| _error.module.js gzip | 3.06 kB | 3.06 kB | ✓ |
| hooks.module.js gzip | 371 B | 371 B | ✓ |
| index.module.js gzip | 212 B | 212 B | ✓ |
| link.module.js gzip | 2.46 kB | 2.46 kB | ✓ |
| routerDirect..dule.js gzip | 273 B | 273 B | ✓ |
| withRouter.m..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 7.22 kB | 7.22 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | Pegase745/next.js example/antd-4-next-9.2 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | zeit/next.js canary  | Pegase745/next.js example/antd-4-next-9.2 | Change |
| - | - | - | - |
| _error.js gzip | 46.5 kB | 46.5 kB | ✓ |
| 404.html gzip | 1.43 kB | 1.43 kB | ✓ |
| hooks.html gzip | 1.07 kB | 1.07 kB | ✓ |
| index.js gzip | 46.5 kB | 46.5 kB | ✓ |
| link.js gzip | 72 kB | 72 kB | ✓ |
| routerDirect.js gzip | 70.2 kB | 70.2 kB | ✓ |
| withRouter.js gzip | 70 kB | 70 kB | ✓ |
| Overall change | 308 kB | 308 kB | ✓ |

</details>

</details>


Commit: df182f60251cdde5af663560e953f56e1e7f2caf

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | zeit/next.js canary  | Pegase745/next.js example/antd-4-next-9.2 | Change |
| - | - | - | - |
| buildDuration | 12.1s | 11.9s | -230ms |
| nodeModulesSize | 52.6 MB | 52.6 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | Pegase745/next.js example/antd-4-next-9.2 | Change |
| - | - | - | - |
| main-HASH.js gzip | 5.12 kB | 5.12 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| 4952ddcd88e7..54d3.js gzip | 4.68 kB | 4.68 kB | ✓ |
| commons.HASH.js gzip | 4.06 kB | 4.06 kB | ✓ |
| de003c3a9d30..b50f.js gzip | 13.6 kB | 13.6 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 67.3 kB | 67.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | Pegase745/next.js example/antd-4-next-9.2 | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.12 kB | 4.12 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| 4952ddcd88e7..dule.js gzip | 5.56 kB | 5.56 kB | ✓ |
| de003c3a9d30..dule.js gzip | 12.4 kB | 12.4 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 61.9 kB | 61.9 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | Pegase745/next.js example/antd-4-next-9.2 | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 4.76 kB | 4.76 kB | ✓ |
| Overall change | 4.76 kB | 4.76 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | Pegase745/next.js example/antd-4-next-9.2 | Change |
| - | - | - | - |
| _app.js gzip | 1.15 kB | 1.15 kB | ✓ |
| _error.js gzip | 4.07 kB | 4.07 kB | ✓ |
| hooks.js gzip | 779 B | 779 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.89 kB | 2.89 kB | ✓ |
| routerDirect.js gzip | 283 B | 283 B | ✓ |
| withRouter.js gzip | 282 B | 282 B | ✓ |
| Overall change | 9.68 kB | 9.68 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | Pegase745/next.js example/antd-4-next-9.2 | Change |
| - | - | - | - |
| _app.module.js gzip | 576 B | 576 B | ✓ |
| _error.module.js gzip | 3.06 kB | 3.06 kB | ✓ |
| hooks.module.js gzip | 371 B | 371 B | ✓ |
| index.module.js gzip | 212 B | 212 B | ✓ |
| link.module.js gzip | 2.46 kB | 2.46 kB | ✓ |
| routerDirect..dule.js gzip | 273 B | 273 B | ✓ |
| withRouter.m..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 7.22 kB | 7.22 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | Pegase745/next.js example/antd-4-next-9.2 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | zeit/next.js canary  | Pegase745/next.js example/antd-4-next-9.2 | Change |
| - | - | - | - |
| index.html gzip | 1.02 kB | 1.02 kB | ✓ |
| link.html gzip | 1.03 kB | 1.03 kB | ✓ |
| withRouter.html gzip | 1.02 kB | 1.02 kB | ✓ |
| Overall change | 3.07 kB | 3.07 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | zeit/next.js canary  | Pegase745/next.js example/antd-4-next-9.2 | Change |
| - | - | - | - |
| buildDuration | 12.4s | 12.3s | -172ms |
| nodeModulesSize | 52.6 MB | 52.6 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | Pegase745/next.js example/antd-4-next-9.2 | Change |
| - | - | - | - |
| main-HASH.js gzip | 5.12 kB | 5.12 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| 4952ddcd88e7..54d3.js gzip | 4.68 kB | 4.68 kB | ✓ |
| commons.HASH.js gzip | 4.06 kB | 4.06 kB | ✓ |
| de003c3a9d30..b50f.js gzip | 13.6 kB | 13.6 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 67.3 kB | 67.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | Pegase745/next.js example/antd-4-next-9.2 | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.12 kB | 4.12 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| 4952ddcd88e7..dule.js gzip | 5.56 kB | 5.56 kB | ✓ |
| de003c3a9d30..dule.js gzip | 12.4 kB | 12.4 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 61.9 kB | 61.9 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | Pegase745/next.js example/antd-4-next-9.2 | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 4.76 kB | 4.76 kB | ✓ |
| Overall change | 4.76 kB | 4.76 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | Pegase745/next.js example/antd-4-next-9.2 | Change |
| - | - | - | - |
| _app.js gzip | 1.15 kB | 1.15 kB | ✓ |
| _error.js gzip | 4.07 kB | 4.07 kB | ✓ |
| hooks.js gzip | 779 B | 779 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.89 kB | 2.89 kB | ✓ |
| routerDirect.js gzip | 283 B | 283 B | ✓ |
| withRouter.js gzip | 282 B | 282 B | ✓ |
| Overall change | 9.68 kB | 9.68 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | Pegase745/next.js example/antd-4-next-9.2 | Change |
| - | - | - | - |
| _app.module.js gzip | 576 B | 576 B | ✓ |
| _error.module.js gzip | 3.06 kB | 3.06 kB | ✓ |
| hooks.module.js gzip | 371 B | 371 B | ✓ |
| index.module.js gzip | 212 B | 212 B | ✓ |
| link.module.js gzip | 2.46 kB | 2.46 kB | ✓ |
| routerDirect..dule.js gzip | 273 B | 273 B | ✓ |
| withRouter.m..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 7.22 kB | 7.22 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | Pegase745/next.js example/antd-4-next-9.2 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | zeit/next.js canary  | Pegase745/next.js example/antd-4-next-9.2 | Change |
| - | - | - | - |
| _error.js gzip | 46.5 kB | 46.5 kB | ✓ |
| 404.html gzip | 1.43 kB | 1.43 kB | ✓ |
| hooks.html gzip | 1.07 kB | 1.07 kB | ✓ |
| index.js gzip | 46.5 kB | 46.5 kB | ✓ |
| link.js gzip | 72 kB | 72 kB | ✓ |
| routerDirect.js gzip | 70.2 kB | 70.2 kB | ✓ |
| withRouter.js gzip | 70 kB | 70 kB | ✓ |
| Overall change | 308 kB | 308 kB | ✓ |

</details>

</details>


Commit: 80cf2b2dec98f1c5b1c3f2c3f1946eeb009a2d8d

File: examples/with-antd-4/src/main.css [hunk 1]
```diff
@@ -0,0 +1 @@
+@import '~antd/dist/antd.css';
```
Comment (subodh-malgonde) [inline examples/with-antd-4/src/main.css]: Can you modify this example so that you do not import the full css bundle?

**Review by Timer — CHANGES_REQUESTED** [review]
We already have an ant-design example. Can you please reopen this PR updating the existing example (which is already ant 4) instead of creating a new one? Thanks!

**Pull request closed without merging**